### PR TITLE
Use weaveworks/flux-get-started repo in docs

### LINKED ISSUE #1360: Consider new Helm example
In #1357 the question came up if we want to continue to use [flux-helm-test](https://github.com/weaveworks/flux-helm-test) as an example. It's referred to from our [Helm tutorial](https://github.com/weaveworks/flux/blob/master/site/helm-get-started.md).

Who has a small example test-case which we could use here?

### PR DESCRIPTION
**Pull request by stefanprodan**
- replace flux-example and flux-helm-test repos with [flux-get-started](https://github.com/weaveworks/flux-get-started)
- rename FluxHelmRelease with HelmRelease
- remove helm-op git flags 
- use the same git URL format everywhere
- update FAQ by removing resolved issues

Fix #1360 #1528

### CODE REVIEW

File: chart/flux/README.md [hunk 1]
```diff
@@ -212,12 +209,6 @@ The following tables lists the configurable parameters of the Weave Flux chart a
 | `helmOperator.chartsSyncInterval` | Interval at which to check for changed charts | `3m`
 | `helmOperator.chartsSyncTimeout` | Timeout when checking for changed charts | `1m`
 | `helmOperator.extraEnvs` | Extra environment variables for the Helm operator pod | `[]`
-| `helmOperator.git.url` | URL of git repo with Helm charts | `git.url`
-| `helmOperator.git.branch` | Branch of git repo to use for Helm charts | `master`
-| `helmOperator.git.chartsPath` | Path within git repo to locate Helm charts (relative path) | `charts`
-| `helmOperator.git.pollInterval` | Period at which to poll git repo for new commits | `git.pollInterval`
-| `helmOperator.git.timeout` | Duration after which git operations time out | `git.timeout`
```
[the remark this answers is not in the document]
Comment (stefanprodan) [reply chart/flux/README.md]: my bad... I put it back

File: chart/flux/README.md [hunk 2]
```diff
@@ -27,24 +27,18 @@ until it can find one.
...
 
- - One repo containing both desired release state information and Charts
-   themselves.
- - Release state information in the form of Custom Resources manifests is
-   located under a particular path ("releaseconfig" by default; can be
-   overriden).
- - Charts are colocated under another path ("charts" by default; can be
-   overriden). Charts are subdirectories under the charts path.
+ - One repo containing desired release state information and Charts themselves.
+ - Charts are co-located under another path or can be referenced from Helm repositories.
```
Comment (squaremo) [inline chart/flux/README.md]:
```suggestion
 - Charts can be co-located with config in the git repo, or be from Helm repositories.
```

File: chart/flux/README.md [hunk 3]
```diff
@@ -27,24 +27,18 @@ until it can find one.
...
 # Git repo
 
- - One repo containing both desired release state information and Charts
-   themselves.
- - Release state information in the form of Custom Resources manifests is
-   located under a particular path ("releaseconfig" by default; can be
-   overriden).
- - Charts are colocated under another path ("charts" by default; can be
-   overriden). Charts are subdirectories under the charts path.
+ - One repo containing desired release state information and Charts themselves.
```
Comment (squaremo) [inline chart/flux/README.md]:
```suggestion
 - One repo containing cluster config (i.e., Kubernetes YAMLs) and zero or more git repos containing Charts themselves.
```

File: site/faq.md [hunk 4]
```diff
@@ -428,22 +418,12 @@ You need to use the `helm delete --purge` option only then Flux will be able rei
 
 ### I've uninstalled Flux and all my Helm releases are gone. Why is that?
 
-On `FluxHelmRelease` CRD deletion, Kubernetes will remove all `FluxHelmRelease` CRs triggering a Helm purge for each release created by Flux.
+On `FluxHelmRelease` CRD deletion, Kubernetes will remove all `HelmRelease` CRs triggering a Helm purge for each release created by Flux.
```
Comment (squaremo) [inline site/faq.md]:
```suggestion
On `HelmRelease` CRD deletion, Kubernetes will remove all `HelmRelease` resources triggering a Helm purge for each release created by Flux.
```

**Pull request merged**